### PR TITLE
v55.3.0

### PR DESCRIPTION
**Pull request by yarneo**

### CODE REVIEW

File: components/TextFields/src/MDCTextInputControllerBase.m [hunk 1]
```diff
@@ -1512,11 +1512,12 @@ - (void)updateLayout {
     return;
   }
 
+  // NOTE: Order of operation here is important.
   [self updateCursor];
-  [self updatePlaceholder];
   [self updateLeadingUnderlineLabel];
   [self updateTrailingUnderlineLabel];
   [self updateTextInput];
+  [self updatePlaceholder];
```
Comment (mohammadcazig) [inline components/TextFields/src/MDCTextInputControllerBase.m]: this should be causing the size change.

File: components/TextFields/src/MDCTextInputControllerFullWidth.m [hunk 2]
```diff
@@ -842,10 +842,11 @@ - (void)updateLayout {
     return;
   }
 
-  [self updatePlaceholder];
+  // NOTE: Order of operation here is important.
   [self updateLeadingUnderlineLabel];
   [self updateTrailingUnderlineLabel];
   [self updateTextInput];
+  [self updatePlaceholder];
```
Comment (mohammadcazig) [inline components/TextFields/src/MDCTextInputControllerFullWidth.m]: this should be causing the size change.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,172 @@
+# 55.3.0
+
+## Breaking changes
+
+## New deprecations
+
+## New features
```
Comment (jverkoey) [inline CHANGELOG.md]: Either fill out or delete these sections.

Comment (jverkoey) [inline CHANGELOG.md]: Flexible header has a new behavioral flag for opting in to better topLayoutGuide support. This is primarily useful when using the flexible header container view controller. To opt in to this new behavior, do the following:

```swift
let container = MDCFlexibleHeaderContainerViewController()
container.isTopLayoutGuideAdjustmentEnabled = true
```

**Review by jverkoey — APPROVED** [review]

**Pull request merged**